Hmm, I don't know why Travis failed, ` arduino:samd` is also installed, it should be able to find the `sam.h`

```
########################################################################
flash_erase.ino: ✖
----------------------------- DEBUG OUTPUT -----------------------------
Picked up JAVA_TOOL_OPTIONS: 
Loading configuration...
Initializing packages...
Preparing boards...
Verifying...
In file included from sketch/flash_erase.ino.cpp:1:0:
/home/travis/.arduino15/packages/adafruit/hardware/samd/1.5.0/cores/arduino/Arduino.h:48:17: fatal error: sam.h: No such file or directory
 #include "sam.h"
                 ^
compilation terminated.
exit status 1
```

--- a/src/Adafruit_QSPI.h
+++ b/src/Adafruit_QSPI.h
@@ -108,7 +108,7 @@ class Adafruit_QSPI
     /// @return true if success
     virtual bool readMemory(uint32_t addr, uint8_t *buffer, uint32_t len) = 0;
 
-    /// Write data to external flash contents, flash sector must be previously erased by \ref eraseSector() first.
+    /// Write data to external flash contents, flash sector must be previously erased first.
     /// Typically it uses quad write command 0x32
     /// @param addr       address to read
     /// @param data       writing data

